proto/ardaos/spv/params.proto

```
@@ -1,15 +1,15 @@
 syntax = "proto3";
-package ardaos.ardaos;
+package ardaos.spv;
 
 import "amino/amino.proto";
```

Buf (1.55.1)

import "amino/amino.proto": file does not exist

(COMPILE)